docs/Makefile

```diff
@@ -3,18 +3,21 @@
 
 # You can set these variables from the command line, and also
 # from the environment for the first two.
-SPHINXOPTS    ?=
+# -W will make the workflow fail if there are any build errors/warnings
+SPHINXOPTS    ?= -W
 SPHINXBUILD   ?= sphinx-build
 SOURCEDIR     = .
 BUILDDIR      = _build
 
 # Put it first so that "make" without argument is like "make help".
 help:
+	rm -rfv "$(SOURCEDIR)/_autosummary"
```

Comment:
That seems strange to perform a destructive action before displaying an help message. I really think this should rather be `clean`:

*(warning: code below contains space instead of the required tabs)*
```
clean: 
    rm -rfv "$(SOURCEDIR)/_autosummary"
    @$(SPHINXBUILD) -M clean "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
```